source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/users/show/index.tsx

```diff
@@ -29,12 +31,14 @@ class User extends React.Component<IProps> {
     const phone = user.phone ? user.phone : t('profile.noPhone');
     const timezone = user.timezone ? `(${user.timezone})` : t('profile.noTimezone');
 
+    const gravatarHash = md5((user.email || '').trim().toLowerCase());
```

Comment:
what does the gravatar look like without an email?

Reply:
Gravatar when not email or bad hash is provided we get a default image. Checkout the param of the url, param d is for default